docs/errors/E0450.md

```diff
@@ -0,0 +1,20 @@
+# E0450: misleading use of comma operator in indexing operation
```

Comment:
This doc title should match the diagnostic's message ("misleading use of ',' operator in in index"). Either change this title or change the diagnostic in diagnostic-types.h.